sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 1.4.0b2 (Unreleased)
-
+- Calling any credential's `get_token` method with no scope raises `ValueError`
```

Comment:
I would rephrase this to start with which method is changed and why it was changed. E.g. 

`TokenCredentials.get_token` requires at least one scope to be provided. While calling the method with no scopes may have worked in earlier versions, the underlying behavior was actually undefined and very likely to change as soon as more tokens were added to the underlying shared cache.

Reply:
I agree with Johan, I would expect the few impacted people to search for this info in the ChangeLog, it's what I do when pytest/requests/etc. is breaking me, looking for the ChangeLog to understand why (before creating an issue). Just seeing "yeah now we raise" is not the best experience we can provide.